src/CoreWCF.RabbitMQ/src/CoreWCF/Channels/RabbitMqConnectionSettings.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Net;
+using RabbitMQ.Client;
+
+namespace CoreWCF.Channels
+{
+    public class RabbitMqConnectionSettings
+    {
+        public const string TempQueuePrefix = "corewcf-temp-";
+        private const string AMQPScheme = "amqp";
```

Comment:
const values are usually private or internal unless it's a value we expect users to consume.